graylog2-web-interface/src/components/common/EntityFilters/FilterConfiguration/SuggestionsList.tsx

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import React, { useState, useCallback } from 'react';
+import debounce from 'lodash/debounce';
+import styled, { css } from 'styled-components';
+
+import { Input, ListGroupItem } from 'components/bootstrap';
+import type { Attribute } from 'stores/PaginationTypes';
+import type { Filters, ValueFilter } from 'components/common/EntityFilters/types';
+import { PaginatedList, NoSearchResult } from 'components/common';
+import useIsKeyHeld from 'hooks/useIsKeyHeld';
+import useFilterValueSuggestions from 'components/common/EntityFilters/hooks/useFilterValueSuggestions';
+import Spinner from 'components/common/Spinner';
+
+const DEFAULT_SEARCH_PARAMS = {
+  query: '',
+  pageSize: 10,
+  page: 1,
+};
+
+const Container = styled.div(({ theme }) => css`
+  color: ${theme.colors.global.textDefault};
+  padding: 3px 10px;
+`);
+
+const SearchInput = styled(Input)`
+  margin-bottom: 6px;
+`;
+
+const StyledListGroup = styled.div`
+  margin-bottom: 0;
+`;
+
+const Hint = styled.div(({ theme }) => css`
+  margin-top: 9px;
+  font-size: ${theme.fonts.size.small};
+`);
+
+type Props = {
+  allActiveFilters: Filters | undefined,
+  attribute: Attribute,
+  filter: ValueFilter | undefined
+  filterValueRenderer: (value: unknown, title: string) => React.ReactNode | undefined,
+  onSubmit: (filter: { title: string, value: string }, closeDropdown: boolean) => void,
+}
+
+const SuggestionsList = ({ attribute, filterValueRenderer, onSubmit, allActiveFilters, filter }: Props) => {
```

Comment:
Can you please add a short description of the suggestions list functionality to the PR description?